qiskit_nature/problems/second_quantization/lattice/models/heisenberg_model.py

```diff
@@ -0,0 +1,131 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Heisenberg model."""
```

Comment:
Please move this file to: `qiskit_nature/second_q/properties/heisenberg_model.py`
It's actual final destination will be in the `hamiltonians` module but it will be easier to refactor alongside with the `FermiHubbardModel` and `IsingModel` once we start work on that :+1: